nat-lab/Pipfile.lock

```diff
@@ -953,6 +953,14 @@
             "markers": "python_version >= '3.8'",
             "version": "==2.32.3"
         },
+        "scapy": {
+            "hashes": [
+                "sha256:5b260c2b754fd8d409ba83ee7aee294ecdbb2c235f9f78fe90bc11cb6e5debc2"
+            ],
+            "index": "pypi",
+            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3' and python_version < '4'",
```

Comment:
Why such python versions? Especially why python 4 is bad?

Reply:
I didn't think of it, just used `pip install scapy` and extracted relevant part into Pipfile.lock